Machine spec validation fails

When trying to use version >= 0.3.1 to validate machine specs, the following error occurs:

```
$ node /app/src/validate.js $(pwd)/qa/machines
internal/modules/cjs/loader.js:968
   throw err;
   ^
 Error: Cannot find module './machines/validator'
 Require stack:
 - /app/src/validate.js
     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:965:15)
     at Function.Module._load (internal/modules/cjs/loader.js:841:27)
     at Module.require (internal/modules/cjs/loader.js:1025:19)
     at require (internal/modules/cjs/helpers.js:72:18)
     at Object.<anonymous> (/app/src/validate.js:8:19)
     at Module._compile (internal/modules/cjs/loader.js:1137:30)
     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
     at Module.load (internal/modules/cjs/loader.js:985:32)
     at Function.Module._load (internal/modules/cjs/loader.js:878:14)
     at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12) {
   code: 'MODULE_NOT_FOUND',
   requireStack: [ '/app/src/validate.js' ]
 }
```

--- a/state-machine/src/validate.js
+++ b/state-machine/src/validate.js
@@ -5,7 +5,7 @@
 
 const fs = require('fs');
 const YAML = require('yamljs');
-const validator = require('./machines/validator');
+const validator = require('@cloudseam/machine-validator');
 
 if (process.argv.length < 3) {
     return exitWithError('Usage: node validate.js spec_directory');

